egs/wsj/s5/steps/segmentation/do_asr_sad_data_dir.sh

```diff
@@ -0,0 +1,257 @@
+#! /bin/bash
+
+# Copyright 2016  Vimal Manohar
+# Apache 2.0.
+
+# This script does nnet3-based speech activity detection given an input 
+# kaldi data directory and outputs a segmented kaldi data directory.
+# This script can also do music detection and other similar segmentation
+# using appropriate options such as --output-name output-music.
+
+set -e 
+set -o pipefail
+set -u
+
+. path.sh
+
+affix=  # Affix for the segmentation
+nj=32
+cmd=queue.pl
+stage=-1
+
+# Feature options (Must match training)
+mfcc_config=conf/mfcc_hires_bp.conf
+feat_affix=bp   # Affix for the type of feature used
+
+convert_data_dir_to_whole=true    # If true, the input data directory is 
+                                  # first converted to whole data directory (i.e. whole recordings)
+                                  # and segmentation is done on that.
+                                  # If false, then the original segments are 
+                                  # retained and they are split into sub-segments.
+
+# Set to true if the test data needs to be downsampled. 
```

Comment:
It will still fail because the compute-mfcc binary reads the sampling-frequency from the config and not the waveform.